HelpSource/Reference/SCDocSyntax.schelp

```diff
@@ -122,6 +122,16 @@ subsection:: Some more info
 ::
 ::
 
+Level 3:
+definitionlist::
+## keyword:: subsubsection\::
+teletype:: SUBSUBSECTION\:: :: name
+|| A named subsubsection. The name should be plain text and can not contain any tags. Example:
+teletype::
+subsubsection:: Some more detailed info
+::
+::
+
 subsection:: Method tags
 These are level 3 subsections:
```

Comment:
Sim. This seems like it would cause ambiguity between methods and subsubsections. What happens if you declare a subsubsection next to a method?

Reply:
No error occurs. 

**Example:** 

I rendered the following modified SinOSc.schelp file:
```
class:: SinOsc
summary:: Interpolating sine wavetable oscillator.
related:: Classes/Osc, Classes/FSinOsc, Classes/SinOscFB, Classes/PMOsc, Classes/Klang
categories::  UGens>Generators>Deterministic


Description::

Generates a sine wave.
Uses a wavetable lookup oscillator with linear interpolation.
Frequency and phase modulation are provided for audio-rate modulation.
Technically, code::SinOsc:: uses the same implementation as  link::Classes/Osc::  except that its table is fixed to be a sine wave made of code::8192:: samples.

subsection:: Other sinewaves oscillators

LIST::
## link::Classes/FSinOsc:: – fast sinewave oscillator
## link::Classes/SinOscFB:: – sinewave with phase feedback
## link::Classes/PMOsc:: – phase modulation sine oscillator
## link::Classes/Klang:: – bank of sinewave oscillators
## link::Classes/DynKlang:: – modulable bank of sinewave oscillators
::


classmethods::

method::ar, kr

argument::freq
Frequency in Hertz.
Sampled at audio-rate.

argument::phase
Phase in radians.
Sampled at audio-rate.
note::phase values should be within the range +-8pi. If your phase values are larger then simply use code::.mod(2pi):: to wrap them.::

argument::mul
Output will be multiplied by this value.

argument::add
This value will be added to the output.

subsubsection::test
This is a test subsubsection.


Examples::

code::

// create an audio-rate sine wave at 200 Hz,
// starting with phase 0 and an amplitude of 0.5
{ SinOsc.ar(200, 0, 0.5) }.play;

// modulate the frequency with an exponential ramp
{ SinOsc.ar(XLine.kr(2000, 200), 0, 0.5) }.play;

// more complex frequency modulation
{ SinOsc.ar(SinOsc.ar(XLine.kr(1, 1000, 9), 0, 200, 800), 0, 0.25) }.play;

// phase modulation (see also PMOsc)
{ SinOsc.ar(800, SinOsc.ar(XLine.kr(1, 1000, 9), 0, 2pi), 0.25) }.play;

::


```
The result is as follows:
![Screenshot 2024-07-05 at 07 42 40](https://github.com/supercollider/supercollider/assets/416281/7608f5a2-6291-44c0-836d-b4e9179e9930)

Let me know if you need more test results!

Reply:
Excellent, sorry away from pc and can't run this.